openapi.yaml

```diff
@@ -1099,6 +1099,9 @@ paths:
                       properties:
                         actor_id:
                           type: string
+                        actor_via_sso:
```

Comment:
what about just calling it `is_sso_user` ?

Reply:
I'd like to keep with the `actor` naming when it's already there. We can think about refactoring the audit log after this goes out.